Packs/GreyNoise/Integrations/GreyNoise/GreyNoise.yml

```
@@ -0,0 +1,409 @@
+category: Data Enrichment & Threat Intelligence
+commonfields:
+  id: GreyNoise
+  version: -1
+configuration:
+- display: API Key
+  name: apikey
+  required: true
+  type: 4
+- additionalinfo: Whether to allow connections without verifying SSL certificates
+    validity.
+  display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+- additionalinfo: Whether to use XSOAR's system proxy settings to connect to the API.
+  display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+description: GreyNoise is a cybersecurity platform that collects and analyzes Internet-wide
+  scan and attack traffic. With this integration, users can contextualize existing
+  alerts, filter false-positives, identify compromised devices, and track emerging
+  threats.
+display: GreyNoise
+name: GreyNoise
+script:
+  commands:
+  - name: ip
+    arguments:
+    - name: ip
+      default: true
+      description: List of IPs.
+      isArray: true
+      required: true
+    description: Runs reputation on IPs.
+    outputs:
+    - contextPath: DBotScore.Indicator
+      description: The indicator that was tested.
+      type: String
+    - contextPath: DBotScore.Score
+      description: The actual score.
+      type: Number
+    - contextPath: DBotScore.Type
+      description: The indicator type.
+      type: String
+    - contextPath: DBotScore.Vendor
+      description: The vendor used to calculate the score.
+      type: String
+    - contextPath: IP.address
+      description: IP address.
+      type: string
+    - contextPath: IP.ASN
+      description: The autonomous system name for the IP address.
+      type: string
+    - contextPath: IP.Hostname
+      description: The hostname that is mapped to IP address.
+      type: string
+    - contextPath: IP.Geo.Country
+      description: The country in which the IP address is located.
+      type: string
+    - contextPath: IP.Geo.Description
+      description: Additional information about the location such as city and region.
+      type: string
+    - contextPath: IP.Malicious.Vendor
+      description: The vendor reporting the IP address as malicious.
+      type: string
+    - contextPath: IP.Malicious.Description
+      description: A description explaining why the IP address was reported as malicious.
+      type: string
+  - arguments:
+    - default: true
+      description: List IP addresses to retrieve quick check about
```

I would consider rephrasing this, for example:
```suggestion
      description: A list of IP addresses to run a quick check on
```